packaging/generic-deb-integrated/debian/ecs.service

```diff
@@ -24,6 +24,7 @@ Restart=on-failure
 RestartSec=10s
 EnvironmentFile=-/var/lib/ecs/ecs.config
 EnvironmentFile=-/etc/ecs/ecs.config
+ExecStartPre=/usr/bin/systemctl -q is-active docker
```

Comment:
`/usr/bin/systemctl` does this path ever change with platform type? 🤔

Reply:
Sync'd offline. We'll remove the ExecStartPre from all of the `ecs.service` files except for the one in `amazon-linux-ami-integrated`.